client/jetpack-cloud/sections/agency-dashboard/controller.tsx

```diff
@@ -23,6 +23,7 @@ export const agencyDashboardContext: Callback = ( context, next ) => {
 	const filter = {
 		issueTypes: issue_types?.split( ',' ),
 		showOnlyFavorites: context.params.filter === 'favorites',
+		showOnlyDevelopmentSites: context.params.filter === 'development',
```

Comment:
This is not being utilized at the moment, I am adding the line to resolve TS issues + this param filter may be needed in the future.